.buildkite/schedules/dependency-analysis.yml

```
@@ -0,0 +1,23 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+---
+
+common_params:
+  # Common plugin settings to use with the `plugins` key.
+  - &common_plugins
+    - automattic/a8c-ci-toolkit#2.15.0
```

Can we use the newest version?
```suggestion
    - automattic/a8c-ci-toolkit#3.4.2
```

---

I would love to do that @wzieba but I really don't want to make a change that can potentially block this PR (ie. failing CI, etc), me not knowing exactly what the diff between the 2 version are. 🤷 

Can I maybe do that on a separate PR next week, and do that for all these 6 libraries I have been (Login, Media Picker) and will be touching next week (Aztec, About, Tracks)? 🙏

---

What works best for you - the only breaking difference is related `nvm` installation, but if you prefer to do this in a separate PR, it's ok to me too 👍

---

Cool, great, I've noted this down and will create separate PRs for each library I have been or will be touching next week. 🗒️ 

> the only breaking difference is related nvm installation

👍

---

FYI @wzieba that this is now done: #150